sequences/__main__.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import pandas as pd
+from keras import Sequential
+from keras.layers import LSTM, Dense
+from matplotlib import pyplot
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.feature_selection import SelectFromModel
+from sklearn.metrics import accuracy_score, confusion_matrix
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+
+
+def predict_sequence():
+    sample = pd.read_csv("../seq/sample.csv")
+    # ressources = pd.read_csv("../seq/res.csv")
+    sample = prepare_data(sample)
+
+    # create series and split in training and validation sets
+    x_series, y_series = create_series(sample, 5, 1)
+    x_train, x_validate, y_train, y_validate = train_test_split(x_series, y_series, test_size=0.2)
```

Comment:
It's usually best to split the data into 3 parts: training, validation and testing. The validation set can then used to tune the model and monitor the training, reserving the test set just for evaluating the results at the end.